doc/man3/SSL_CTX_set_cert_verify_callback.pod

```diff
@@ -63,6 +63,11 @@ on resumption, even though no chain is presented int that case.
 Moreover, the calling application will be informed about the detailed result of
 the verification procedure and may elect to base further decisions on it.
 
+I<callback> may call L<X509_verify_cert(3)> to run the built-in verification
+function. This may be useful if application wishes to dynamically reconfigure
+I<x509_store_ctx> before verification, or postprocess the result. In this case,
+L<X509_verify_cert(3)> will set the B<error> member as described above.
```

Comment:
This is useful, thanks!
Should there be mention of the fact that on retry a fresh `X509_STORE_CTX` is initialised and used, and any state from a previous incomplete attempt is discarded?